resources/process_workflow.md

```diff
@@ -35,6 +35,19 @@ Be sure not to create a task such as "create a input field", or "create an API",
 
 All SMART tasks should be linked to a User Story, which should be properly detailed with its goal (it can be written using the User Story pattern), Acceptance criteria, and BDD if necessary.
 
+We are following Gherkin BD workflow to describe our User Stories, so, we have a template in ClubHouse to be used:
+
+*    Title: [Verb] [What] [Where]:
```

Comment:
**suggestion:** add markdown in code format:

```
<!-- Template markdown here -->
```

Reply:
Done in b6c62c3